acceptance/config/aio/options.rb

```diff
@@ -1,6 +1,7 @@
 {
   :type => 'aio',
   :is_puppetserver => true,
+  :puppetserver_confdir => '/etc/puppetlabs/puppetserver/conf.d',
```

Comment:
ah, this actually need to be  `:'puppetserver-confdir' => '/etc/puppetlabs/puppetserver/conf.d',` note the dash, which has to be quoted since it's a symbol. I think it was passing for you because the version of beaker we are pinned to already set that as the AIO default.

Let's hold off on this until we get puppet testing against beaker#master